hw2/index.js

```diff
@@ -1,8 +1,8 @@
 function NamedOne (firstName, lastName) {
 
-    let res = {
+    const res = {
 
-        firstName: lastName,
+        firstName: firstName,
```

Comment:
@matveyprupas Last thing here: to make it shorter you can use just `firstName` instead of `firstName: firstName`. When your field has the same name as your variable it's better to reduce your code

Reply:
Done in next commit